### PR TITLE
Fix some missing qdel hints

### PR DESCRIPTION
**Pull request by Cyberboss**

### CODE REVIEW

**Comment by MrStonedOne** [comment]
orbit's datum has one

**Comment by optimumtact** [comment]
@MrStonedOne can you clarify your comment

File: code/modules/orbit/orbit.dm [hunk 1]
```diff
@@ -33,7 +33,7 @@
 			if (!orbiting.orbiters.len)//we are the last orbit, delete the list
 				orbiting.orbiters = null
 		orbiting = null
-	..()
+	return ..()
```
Comment (Jordie0608) [inline code/modules/orbit/orbit.dm]: This @optimumtact

Comment (optimumtact) [inline code/modules/orbit/orbit.dm]: yeah but what is the context, that value wasnt' being returned before

Comment (Jordie0608) [inline code/modules/orbit/orbit.dm]: I think it was causing `QDEL_HINT_QUEUE` to not get passed all the way to when `qdel(orbiting)` is called in `stop_orbit()`

**Pull request merged**